cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -168,6 +168,9 @@ IS NODE KEY
 | Drop a node key constraint without specifying a name.
 |===
 
+[NOTE]
+The `IF [NOT] EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards.
```

Comment:
Maybe add the note in `... only if it does not already exist` and `drop a non-existing constraint` as well?